web/web.go

```diff
@@ -57,6 +57,7 @@ type ConfigResponse struct {
 	AlgoliaDocsIndexName     string          `json:"algolia_docs_index_name"`
 	AlgoliaDraftsIndexName   string          `json:"algolia_drafts_index_name"`
 	AlgoliaInternalIndexName string          `json:"algolia_internal_index_name"`
+	BypassGoogleAuth         bool            `json:"bypass_google_auth"`
```

Comment:
This is a nit, but I wonder if we could use "skip" instead of "bypass"? For example, `SkipGoogleAuth`, `skip_google_auth`, `SKIP_GOOGLE_AUTH `?

Reply:
Done! (I didn't have any preference on this wording)